src/main/resources/application.yaml

```diff
@@ -29,6 +29,10 @@ core_case_data:
   api:
     url: 'http://localhost:4452'
 
+document_management:
+  documents:
+    url: ${DOCUMENT_MANAGEMENT_URL:http://localhost:4460}/documents
```

Comment:
just a thought, since recently looked into shared infra for local ccd setup. should this be `dm-store` instead of `localhost`?

Reply:
I think it should be `dm-store`, but not sure. we have the same in functional test properties.  @nitinprabhu do you know what this value should be?

Reply:
can be localhost yes, didn't consider the scope. i'll resolve this thread